src/languages/sqf.js

```diff
@@ -1079,24 +1165,32 @@ export default function(hljs) {
     'getSubtitleOptions',
     'getSuppression',
     'getTerrainGrid',
+    'getTerrainHeight',
     'getTerrainHeightASL',
+    'getTerrainInfo',
     'getText',
     'getTextRaw',
+    'getTextureInfo',
     'getTextWidth',
+    'getTiParameters',
     'getTotalDLCUsageTime',
     'getTrimOffsetRTD',
+    'getTurretLimits',
+    'getTurretOpticsMode',
+    'getUnitFreefallInfo',
     'getUnitLoadout',
     'getUnitTrait',
+    'getUnloadInCombat',
+    'getUserInfo',
     'getUserMFDText',
     'getUserMFDValue',
     'getVariable',
     'getVehicleCargo',
-    'getVehicleTIPars',
+    'getVehicleTiPars',
```

Comment:
```suggestion
    'getVehicleTIPars',
```
https://community.bistudio.com/wiki/getVehicleTIPars